lib/src/Base/Func/openturns/ExpertMixture.hxx

```diff
@@ -47,7 +47,8 @@ public:
 
   /** Constructor */
   ExpertMixture(const Basis & experts,
-                const Classifier & classifier);
+                const Classifier & classifier,
+                const Bool supervised = true);
 
```

Comment:
yes I mean only at initialization
At this level we had various incoherence in the library (sometimes we get RM keys, sometimes no)